### DISCUSSION TITLE
Chocolatey vs Winget

### DISCUSSION THREAD

**Discussion by NameOfTheDragon** [supermerill/SuperSlicer#4020] [General]
I see that SS has started supporting install via Chocolatey, and a couple of years ago I would have welcomed that, but Chocolatey is a bit "last year" now. It has gone a bit corporate and hides some of its features behind a paywall, which I think is why Microsoft made `winget` and built it into Windows.

SuperSlicer is looking a bit behind the curve then, since other slicers are already supported via winget. For example, you can:

```
winget install Prusa3D.PrusaSlicer
```

and walla. Much less fuss than Chocolatey, actually, and no additional tools to install because `winget` is part of windows.

Here are the supported packages based on the keyword "slicer":

```
Microsoft Windows [Version 10.0.22621.2861]
(c) Microsoft Corporation. All rights reserved.

C:\Users\Tim> winget search slicer

Name              Id                        Version   Match          Source
---------------------------------------------------------------------------
UVtools           PTRTECH.UVtools           4.0.5     Tag: slicer    winget
PrusaSlicer       Prusa3D.PrusaSlicer       2.7.1     Tag: slicer    winget
OrcaSlicer        SoftFever.OrcaSlicer      1.8.1                    winget
PrusaSlicer Alpha Prusa3D.PrusaSlicer.Alpha 2.6.0-rc2                winget
Ultimaker Cura    Ultimaker.Cura            5.6.0     Tag: 3d-slicer winget
```
I think this is the right way to go for SuperSlicer, too, and I hope you will agree @supermerill  and make it available via `winget`.

**Comment by supermerill** [comment]
thanks for tip
I'm not maintaining any repository myself, chocolatey, and any linux thingy has been done by someone else. 
I'll try to maintain it or maybe i can do a script.

**Comment by Anders0lesen** [comment]
Mind if I take a stab at getting it into `winget`? I was just now installing it and I was surprised that it wasn't in the catalog

**Reply by supermerill** [reply]
you can
If I only have to merge somtehing that works, It's much easier.
